.changes/Added-20260825-final-publication-plan.yaml

```
@@ -0,0 +1,3 @@
+kind: Added
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)